module/VuFind/src/VuFind/Config/Handler/Ini.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/**
+ * Ini config handler.
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2025.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Config_Handlers
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @author   Thomas Wagener <wagener@hebis.uni-frankfurt.de>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+
+namespace VuFind\Config\Handler;
+
+use VuFind\Config\Location\ConfigLocationInterface;
+
+use function in_array;
+use function is_array;
+
+/**
+ * Ini config handler.
+ *
+ * @category VuFind
+ * @package  Config_Handlers
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @author   Thomas Wagener <wagener@hebis.uni-frankfurt.de>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+class Ini extends AbstractBase
+{
+    /**
+     * Parses the configuration in a config location.
+     *
+     * @param ConfigLocationInterface $configLocation Config location
+     *
+     * @return array
+     */
+    public function parseConfig(ConfigLocationInterface $configLocation): array
+    {
+        $data = parse_ini_file($configLocation->getPath(), true);
+        if ($data === false) {
+            return [];
```

Comment:
Should we possibly log a warning or throw an error if an .ini file fails to parse, or is there a reason we want to swallow problems at this stage?